### PR TITLE
Iow 41 - Support running the graph server off a path on the ALB

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [X] Update the changelog appropriately

Title
-----------
Support running the graph server off a path on the ALB

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: graph-server/src/index.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const renderToRespone = require('./renderer');
 
 const PORT = process.env.NODE_PORT || 2929;
 const CACHE_TIMEOUT = 15 * 60;  // 15 minutes
+const PATH_CONTEXT = process.env.NODE_PORT || '/api/graph-images';
```
Comment (ayan-usgs) [inline graph-server/src/index.js]: So, I could either use the default or I could pass in a `NODE_PORT` environment variable to set the context. Am I understanding that right?

Comment (mbucknell) [reply graph-server/src/index.js]: Potentially but at the moment I don't think either the docker file or the npm script has been set up to use that. That's also a typo that I have fixed. Should be PATH_CONTEXT. But regardless, I think some changes would need to be made in Dockerfile and package.json to use the environment variables.

Comment (ayan-usgs) [reply graph-server/src/index.js]: >I think some changes would need to be made in Dockerfile and package.json to use the environment variables.

Yep. Thought we were using one our images, but we're actually using a clean version of Alpine off Docker Hub.

Comment (ayan-usgs) [reply graph-server/src/index.js]: Suffice it to say, I think this achieves what we set out to do.

**Pull request merged**